src/main/java/com/google/codeu/data/Country.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.data;
+
+public class Country {
+  private String code;
+  private String name;
+  private double lat;
+  private double lng;
+
+  public Country(String code, String name, double lat, double lng) {
```

Comment:
I think this is still up for discussion, but how do we plan on tying in the subreddits to the Country class? The Country page should display the subreddits and several of the most recent posts, so maybe we can add a field to the Country class that keeps track of these channels somehow.

Reply:
I think that should be done soon, but also I think it's out of the scope of this particular PR. 

My goal with this PR was to get something to build off of - the more features and functionality I am trying to implement, the longer it will take; I see the country pages as being vital to much of the functionality of most other things, so I want to get this completed, and wecan add this display functionality soon